frontend-react/src/components/header/SignInOrUser.tsx

```diff
@@ -37,7 +37,7 @@ const SignInOrUser = () => {
     ) : (
         <NavLink to="/daily-data">
             <Button type="button" inverse={isPreview}>
-                Log in {isPreview ? "via OtkaPreview" : ""}
+                Log in {isPreview ? "via OktaPreview" : ""}
```

Comment:
i have seen this a thousand times in the app and never saw it. even now it took me about 10-15 seconds to see the difference! 🤣 Good eye, @mreifman !